pocket_coffea/workflows/genweights.py

```diff
@@ -48,4 +94,13 @@ def process(self, events: ak.Array):
             if self._hasSubsamples:
                 raise Exception("This processor cannot compute the sum of genweights of subsamples.")
 
+        self.process_extra_before_skim()
+        self.skim_events()
```

Comment:
This call should be moved to line 104. No need to skim in the genweights processor if not for saving the skimmed files.